test/gateway/ost_prime/increase_supply.js

```diff
@@ -191,8 +179,8 @@ contract('OSTPrime.increaseSupply()', function (accounts) {
 
     assert.strictEqual(
       eventData._to,
-      beneficiary,
-      `The _to address in the event should be equal to ${beneficiary}.`,
+      ostPrime.address,
+      `The _to address in the event should be equal to ${ostPrime.address}.`,
```

Comment:
Why is that? 🤔

Reply:
In OST prime on increase ERC20 balance is give to OST prime contract on behalf of that it provides basetoken to beneficiary.

Reply:
Ah right. Is that documented on the function?